vagrant/provisioning/resize-disk.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo ", +" | sudo sfdisk --force -N 3 /dev/vda
```

Comment:
Maybe also add short comment what this step does?

Reply:
as, I commented, we will unify, this anyway, it doesn't help to modify each single copy of it. I will add a comment to the ticket for the unification.